pkgs/tools/package-management/akku/overrides.nix

```diff
@@ -17,18 +17,27 @@ in
 {
   chez-srfi = joinOverrides [
     (pkg: old: {
+      version = "0.0.0-unstable-2024-10-31";
       preCheck = ''
         SKIP='
         multi-dimensional-arrays.sps
         time.sps
         tables-test.ikarus.sps
         lazy.sps
+        pipeline-operators.sps
         '
       '';
+      src = fetchFromGitHub {
+        owner = "arcfide";
+        repo = "chez-srfi";
+        rev = "b424440b94037ace50fc8179f159b7e4c53e59e6";
+        hash = "sha256-Qg8XveLObiD6bZJ3FQNAgbZepnQHcB2GC+wzh58I5mA=";
+      };
+      unpackPhase = "";
```

Comment:
Does `dontUnpack = true` work?

Reply:
I didn't cross my mind. It'd definitely be cleaner, but the build directory seems completely empty, so no SRFIs get installed. I suppose `dontUnpack = true` somehow interferes with `fetchFromGitHub`?

Directory during `preCheck` with `dontUnpack = true`:
```
> drwx------ 3 nixbld nixbld  100 Nov 14 17:26 .
> drwxr-x--- 9 nobody nixbld 4096 Nov 14 17:26 ..
> drwxr-xr-x 4 nixbld nixbld  140 Nov 14 17:26 .akku
> -rw-r--r-- 1 nixbld nixbld  156 Nov 14 17:26 Akku.lock
> -rw------- 1 nixbld nixbld 8450 Nov 14 17:26 env-vars
```

Directory during `preCheck` with `unpackPhase = ""`:
```
> -rw-r--r--  1 nixbld nixbld  145 Jan  1  1970 %3a8.sls
> drwxr-xr-x  2 nixbld nixbld   60 Jan  1  1970 %3a9
> -rw-r--r--  1 nixbld nixbld  156 Jan  1  1970 %3a9.sls
> drwxr-xr-x  2 nixbld nixbld  160 Jan  1  1970 %3a98
> -rw-r--r--  1 nixbld nixbld  211 Jan  1  1970 %3a98.sls
> drwxr-xr-x  3 nixbld nixbld   80 Jan  1  1970 %3a99
> -rw-r--r--  1 nixbld nixbld  373 Jan  1  1970 %3a99.sls
...
```

Reply:
Then it's not worth it. We can leave it as is.